<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>934</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="GKChess::UI::BoardEdit" name="boardView" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Set Light Square Color</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Set Dark Square Color</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>Set Light Piece Color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>Set Dark Piece Color</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Set Piece Size:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="sldr_pieceSize">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Set Square Size:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="sldr_squareSize">
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Set Border Size:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="sldr_borderSize">
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBelow</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Dump to File</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GKChess::UI::BoardEdit</class>
   <extends>QWidget</extends>
   <header>gkchess_boardedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>_change_light_color()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>595</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>_change_dark_color()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>595</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>_dump_to_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>770</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>_change_light_piece_color()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>570</x>
     <y>595</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldr_pieceSize</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>_change_piece_size(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>648</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldr_squareSize</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>_change_square_size(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>692</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>476</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sldr_borderSize</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>_change_border_thickness(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>741</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>493</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>_change_dark_piece_color()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>641</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>608</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_change_light_color()</slot>
  <slot>_change_dark_color()</slot>
  <slot>_dump_to_file()</slot>
  <slot>_change_piece_size(int)</slot>
  <slot>_change_square_size(int)</slot>
  <slot>_change_border_thickness(int)</slot>
  <slot>_change_light_piece_color()</slot>
  <slot>_change_dark_piece_color()</slot>
 </slots>
</ui>
